pkg/k8s/pod/security.go

```diff
@@ -85,3 +86,11 @@ type WindowsOptions struct {
 	GMSACredentialSpec     string
 	RunAsUserName          string
 }
+
+func (wo *WindowsOptions) toK8S() *v1.WindowsSecurityContextOptions {
```

Comment:
should return nil if nothing is configured

Reply:
done